justfile

```diff
@@ -54,7 +62,7 @@ frb-integrate:
 # set up the CI environment for the app
 setup-ci:
     curl -L --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/cargo-bins/cargo-binstall/main/install-from-binstall-release.sh | bash
-    cargo binstall -y flutter_rust_bridge_codegen@2.7.0 cargo-expand
+    cargo binstall -y flutter_rust_bridge_codegen@2.7.1 cargo-expand
```

Comment:
This line can be moved to the check frb job, because now this is the only place we use the frb codegen.